docs/faq/proof-of-stake/general.md

```diff
@@ -26,25 +20,7 @@ The ticket price is always refunded, no matter if your ticket votes, misses or e
 
 ---
 
-#### 4. What are fees?
-
-The PoS system utilizes two types of fees, a "txfee" (also known as a 'split' fee) and a "ticketfee".
-
-The txfee is a fee you pay the network for handling the transaction to buy your ticket. This fee is at default set to a low amount (0.0001 DCR/kB) and should not be changed.
-The ticketfee is the fee you pay if your ticket gets mined. This fee is an incentive for PoW miners to mine your ticket and add it to the ticket pool.
-
-In case the buying window is full, meaning there are more tickets being bought than there are available to be mined during that window, the ticketfee will be used to determine which tickets get put into
-the ticket pool. The tickets with the highest ticketfee will be selected by the PoW miners.
-
-Fees are calculated in DCR per kilobyte. Since a ticket is only about 300 bytes (for solo) or 540 bytes (for VSP) in size, the actual fees you pay will be lower than what you set as the specific fee amount.
```

Comment:
We are losing the 540 bytes figure in this PR. Can we include it on the overview page where we mention that a solo purchase is 300 bytes?

Reply:
Good catch. When incorporating the 540 bytes figure though, did realize that the math was a bit fuzzy on our current example on the overview page. We say that the 300 Bytes solo transaction will cost 0.1 DCR, but 0.3 DCR/kB X 300 Bytes = 0.09 DCR (assuming 1,000 bytes/kilobyte, not 1,024). Likewise 540 DCR using this equation would give a cost of 0.162 DCR. Assuming my math is correct, have updated text to:

> Therefore, with a higher transaction size, you will end up paying a higher absolute fee. For example, solo-staking ticket purchases consume around 300 Bytes, whereas staking with a VSP will consume around 540 Bytes. Therefore, with a a **Ticket Fee** of 0.3 DCR/kB, a solo-staked ticket would cost ~0.09 DCR, and a VSP-staked ticket would cost ~0.16 DCR. 

Also, removed the following sentence about the fee being refunded if the ticket isn't mined. It was a bit clunky, and we also provide that info in the Ticket Lifecycle section.

Reply:
We discussed on Matrix to remove the 0.3 altogether and just refer to the actual fee of 0.0001

Reply:
[Discussed](https://matrix.to/#/!dhHYPTtCtvPSUfTepT:decred.org/$155604517624183TYTTU:decred.org?via=decred.org&via=matrix.org&via=zettaport.com) this a bit more in #support just to clarify for myself. This is the text we came up with:

>The **Ticket Fee** defaults to the minimum (0.0001 DCR/kB), which is typically sufficient.

Reply:
I like the new text, but just want to highlight that we are now losing the estimated tx size completely. 

I dont think the 300/540 bytes figures are included anywhere else in the docs, so it might be nice to keep them somewhere.

Reply:
So, I was able to find these numbers on the [buying tickets](https://docs.decred.org/wallets/cli/dcrwallet-tickets/) page for the CLI Wallet. Since you can only do solo voting using the CLI wallet, I think this is OK. I was trying to work this back in on this page, but couldn't think of a non-awkward place as the concepts of solo voting and VSP aren't introduced until the end of the page.

Reply:
Sounds good, lets go with it then. Thanks for checking